plugins/inputs/nfsclient/nfsclient.go

```diff
@@ -293,9 +298,14 @@ func (n *NFSClient) processText(scanner *bufio.Scanner, acc telegraf.Accumulator
 		}
 
 		if !skip {
-			n.parseStat(mount, export, version, line, acc)
+			err := n.parseStat(mount, export, version, line, acc)
+			if err != nil {
+				return err
```

Comment:
The error returns in various places here could use some more context. example:
```suggestion
				return fmt.Errorf("could not parseStat: %w", err)
```